test-suites/gutenberg/basic-test-suite.md

```diff
@@ -0,0 +1,50 @@
+# Basic Test Suite
+
+This suite groups holds the different test cases that are considered the minimum coverage when testing the editor:
```

Comment:
```suggestion
This suite groups the different test cases that are considered the minimum coverage when testing the editor:
```

Reply:
Oh, that's a typo. Thanks for calling it out, I'll update it.